ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -28,15 +28,16 @@
 #import "ADCustomHeaderHandler.h"
 #import "ADTelemetryUIEvent.h"
 #import "ADTelemetryEventStrings.h"
+#import "ADURLSessionDemux.h"
 
-static NSMutableDictionary* s_handlers = nil;
-static NSString* s_endURL = nil;
-static ADTelemetryUIEvent* s_telemetryEvent = nil;
 
-static NSString* kADURLProtocolPropertyKey = @"ADURLProtocol";
+static NSMutableDictionary *s_handlers      = nil;
+static NSString *s_endURL                   = nil;
+static ADTelemetryUIEvent *s_telemetryEvent = nil;
 
+static NSString *s_kADURLProtocolPropertyKey  = @"ADURLProtocol";
 
-static NSUUID * _reqCorId(NSURLRequest* request)
+static NSUUID *_reqCorId(NSURLRequest* request)
```

Comment:
And what exactly do you gain from using a class method instead of a static helper function?